### PR TITLE
updating the sort and import

### PR DESCRIPTION
**Pull request by AnushkaKhare-Eng**
# Overview

What is the high-level goal of this PR? Is there any important context we should have?

# Changes

List the changes you've made.

# Questions & Notes

Are there any open questions? Anything else important we should know?

# Issue tracking

List any relevant work that still needs to be done. Open and link to issues for them if applicable.

Does this close any tasks?

# Testing & Demo

How did you test these changes? How can the reviewer test these changes?

If the changes are to the UI show screenshots, GIFs, and/or videos.

### CODE REVIEW

File: www/.eslintrc.js [hunk 1]
```diff
@@ -12,9 +12,11 @@ module.exports = {
         ecmaVersion: 12,
         sourceType: 'module',
     },
-    plugins: ['react', '@typescript-eslint', 'prettier'],
+    plugins: ['react', '@typescript-eslint', 'prettier', 'simple-import-sort'],
```
Comment (calebschoepp) [inline www/.eslintrc.js:15]: Good job adding the plugin!

**Review by calebschoepp — APPROVED** [review]
Great work Anushka. Fun working through this with you. LGTM!

**Review by calebschoepp — APPROVED** [review]
FIXED :smile:

**Comment by cyrusdiego** [comment]
Not sure if this is something you're already aware of, but it looks like the CI is failing because all the files haven't been formatted yet. Can you add the following line into the package.json in `www` directory please then run `npm run lint:fix` in the `www` directory to format the imports. 

```
"lint:fix": "eslint src --ext .js,.ts --max-warnings 0 --fix",
```
this line should be in the "scripts" section of the package.json file. And can you update the README.md so that it has the command as part of the available commands. Refer to [this](https://github.com/UAlbertaCompEClub/compe-plus/pull/59) as a guide for where the README and package.json needs to be updated. 

I am also free this weekend if you want to hop on a call and we can walk through this as well :)

**Pull request merged**